### PR TITLE
update catalan translations

### PR DESCRIPTION
**Pull request by capitantrueno**

### CODE REVIEW

File: po/ca.po [hunk 1]
```diff
@@ -92,13 +108,20 @@ msgid ""
...
+"no se guarda cap dada d'ús: no tinc cap idea de què féu amb "
+"Ephemeral, i no tinc cap interés en saber-ho!"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:24
 msgid ""
 "Ephemeral uses Startpage.com—the world's most private search engine—by "
 "default to avoid as much Google tracking as possible. You can also choose "
 "DuckDuckGo from the menu if that’s your preference."
 msgstr ""
+"Ephemeral usa Startpage.com -el motor de cerca més privat del món- per "
```
Comment (fitojb) [inline po/ca.po]: * Use — instead of -
* The word _per_ is written twice

File: po/ca.po [hunk 2]
```diff
@@ -107,13 +130,21 @@ msgid ""
 "passwords? Ephemeral has you covered: just hit your other browser’s icon in "
 "the header and the current page is opened up there."
 msgstr ""
+"Què passa si us trobeu amb un problema de compatibilitat per culpa de "
+"l'antirrastreig, o si voleu accedur a un lloc web amb la contrasenya que vau guardar a "
+"un altre navegador? Ephemeral té la solució: pulseu la icona del seu "
```
Comment (fitojb) [inline po/ca.po]: _prémer_ instead of _pulsar_ also here

File: po/ca.po [hunk 3]
```diff
@@ -122,31 +153,35 @@ msgid ""
...
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:28
 #: src/Views/WelcomeView.vala:34
 msgid "For the best protection, always use a VPN."
-msgstr ""
+msgstr "Per la millor protección, utilitzeu sempre una VPN."
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:36
 msgid "All New Search + Suggestions"
-msgstr ""
+msgstr "Totes les noves cerques + Sugerències"
```
Comment (fitojb) [inline po/ca.po]: _All-new search_ means “cerca completament nova”

File: po/ca.po [hunk 4]
```diff
@@ -156,358 +191,372 @@ msgid ""
...
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:74
 msgid "The last-used browser is remembered for next time"
-msgstr ""
+msgstr "L'últim navegador utilitzat es recordarà per la propera vegada"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:75
 msgid "New \"Reset Preferences\" menu item and dialog"
-msgstr ""
+msgstr "Nou menu i diàleg \"Restableix les preferències\""
```
Comment (fitojb) [inline po/ca.po]: * _Restableix → Reinicialitza_
* Use guillemets «»

File: po/ca.po [hunk 5]
```diff
@@ -156,358 +191,372 @@ msgid ""
...
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:85
 msgid "Hallo! Dutch translations thanks to Heimen Stoffels"
-msgstr ""
+msgstr "Hallo! Traduccions holandeses gràcies a Heimen Stoffels"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:91
 msgid "Styling fixes for non-native platforms."
-msgstr ""
+msgstr "Correccions d'estil per plataformes no natives."
```
Comment (fitojb) [inline po/ca.po]: Per → per a

File: po/ca.po [hunk 6]
```diff
@@ -156,358 +191,372 @@ msgid ""
...
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:96
 msgid "Fixed a few small issues with zoom:"
-msgstr ""
+msgstr "Corregits uns problemes amb el zoom:"
```
Comment (fitojb) [inline po/ca.po]: I’d use _escala_ or _ampliació_ instead of _zoom_

File: po/ca.po [hunk 7]
```diff
@@ -156,358 +191,372 @@ msgid ""
...
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:98
 msgid "Zoom controls are now only sensitive when the web view is visible"
-msgstr ""
+msgstr "Els controls de zoom sols estan actius quan la web és visible"
```
Comment (fitojb) [inline po/ca.po]: Same here, and in the following couple of strings

File: po/ca.po [hunk 8]
```diff
@@ -156,358 +191,372 @@ msgid ""
...
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:109
 msgid "Zoom in, out, and default from the new menu or with the keyboard"
-msgstr ""
+msgstr "Augmenteu el zoom, disminuiu-ho i restabliu-ho des del nou menú o amb el teclat"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:110
 msgid "New Window action moved into the menu"
-msgstr ""
+msgstr "S'ha mogut l'acció Nova Finestra dins del menú"
```
Comment (fitojb) [inline po/ca.po]: Nova Finestra → Finestra nova (no Title Case please)

File: po/ca.po [hunk 9]
```diff
@@ -156,358 +191,372 @@ msgid ""
...
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:111
 msgid "Added a Quit All Windows action to the menu"
-msgstr ""
+msgstr "S'ha afegit l'acció Tancar totes les finestres al menu"
```
Comment (fitojb) [inline po/ca.po]: _Tancar → Tanca_

File: po/ca.po [hunk 10]
```diff
@@ -156,358 +191,372 @@ msgid ""
...
-msgstr ""
+msgstr "Nous estils per mantindre les coses a la moda, i a més:"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:136
 msgid ""
 "The new start page makes Ephemeral faster and makes it not touch the network "
 "until you're ready to browse"
 msgstr ""
+"La nova pàgina d'inici fa Ephemeral més ràpid i que no toque la xarxa "
+"fins que estigueu preparats per navegar"
```
Comment (fitojb) [inline po/ca.po]: _preparats → preparat_ (remember it’s _vós_ and not _vosaltres_ :smiley_cat: )

File: po/ca.po [hunk 11]
```diff
@@ -156,358 +191,372 @@ msgid ""
...
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:137
 msgid "Middle- or Ctrl-click a link to open it in a new window"
-msgstr ""
+msgstr "Féu Ctrl-click o click amb el botó del mig per obrir un enllaç a una nova finestra"
```
Comment (fitojb) [inline po/ca.po]: _click → clic_

File: po/ca.po [hunk 12]
```diff
@@ -156,358 +191,372 @@ msgid ""
...
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:138
 msgid ""
 "Displayed URLs are kept in sync for pages that manually update the URL "
 "without a page load"
 msgstr ""
+"Les URLs mostrades es mantenen sincronitzades per aquelles pàgines que actualizen manualment "
```
Comment (fitojb) [inline po/ca.po]: _URL_ is masculine (_Llocalitza**dor** de recurs uniform_) and takes no plural _s_ as any other acronym: “_Els URL mostrats…_”

File: po/ca.po [hunk 13]
```diff
@@ -156,358 +191,372 @@ msgid ""
...
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:139
 msgid "Local HTML files can now be opened"
-msgstr ""
+msgstr "Ara es poden obrir els fitxers HTML locals"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:146
 msgid "More informative \"set as default\" infobar"
-msgstr ""
+msgstr "Barra \"establiu per defecte\" més informativa"
```
Comment (fitojb) [inline po/ca.po]: `«»` instead of `"`

File: po/ca.po [hunk 14]
```diff
@@ -156,358 +191,372 @@ msgid ""
...
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:147
 msgid "Tweaked \"set as default\" infobar styling"
-msgstr ""
+msgstr "S'ha modifica l'estil de la barra \"establiu per defecte\""
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:154
 msgid "Search from URL entry!"
-msgstr ""
+msgstr "Cerquey des de l'entrada d'URL!"
```
Comment (fitojb) [inline po/ca.po]: Little slip of the finger here (I myself am very prone to this typo :stuck_out_tongue: )

File: po/ca.po [hunk 15]
```diff
@@ -156,358 +191,372 @@ msgid ""
...
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:155
 msgid "Asks (politely!) to be set as the default browser"
-msgstr ""
+msgstr "Demana (educadament!) per que l'establiu com el navegador per defecte"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:156
 msgid "Open unsupported links (like AppCenter ones) in the OS-default app"
-msgstr ""
+msgstr "Obriu enllaços no soportats (com els de l'AppCenter) a l'aplicació per defecte del sistema"
```
Comment (fitojb) [inline po/ca.po]: _soportats_ → _admesos_

File: po/ca.po [hunk 16]
```diff
@@ -156,358 +191,372 @@ msgid ""
...
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:163
 msgid "Add \"New Window\" to headerbar, app launcher, and keyboard shortcuts"
-msgstr ""
+msgstr "S'ha afegit \"Nova Finestra\" a la barra de capçalera, al llançador i a les dreceres de teclat"
```
Comment (fitojb) [inline po/ca.po]: _Finestra nova_ and `«»`

File: po/ca.po [hunk 17]
```diff
@@ -156,358 +191,372 @@ msgid ""
...
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:164
 msgid "Collapse external browsers into a menu when there's more than one"
-msgstr ""
+msgstr "Comprimits els navegadors externs a un menú quan hi ha més d´'un"
```
Comment (fitojb) [inline po/ca.po]: Remove the extraneous grave-accent character

File: po/ca.po [hunk 18]
```diff
@@ -156,358 +191,372 @@ msgid ""
...
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:170
 msgid "Add several useful keyboard and mouse shortcuts"
-msgstr ""
+msgstr "Afegits algunes dreceres útils de teclat i ratolí"
```
Comment (fitojb) [inline po/ca.po]: _Afegits → Afegides_

File: po/ca.po [hunk 19]
```diff
@@ -156,358 +191,372 @@ msgid ""
...
 
 #: data/launcher.desktop.in:12
 msgid "WWW;web;browser;internet;private;incognito;focus;temporary;cookies;"
-msgstr ""
+msgstr "WWW;web;navegador;internet;privadesa;incògnit;focus;temporal;cookies;"
 
 #: data/launcher.desktop.in:17
 msgid "New Window"
-msgstr ""
+msgstr "Nova Finestra"
```
Comment (fitojb) [inline po/ca.po]: _Finestra nova_ (adjectives follow nouns; no Title Case pls)

File: po/ca.po [hunk 20]
```diff
@@ -156,358 +191,372 @@ msgid ""
...
 
 #: src/MainWindow.vala:90
 msgid "Stop loading"
-msgstr ""
+msgstr "Para la càrrega"
 
 #: src/MainWindow.vala:101
 msgid "Erase browsing history"
-msgstr ""
+msgstr "Borra l'historial de navagació"
```
Comment (fitojb) [inline po/ca.po]: _Borra_ → _Neteja_

File: po/ca.po [hunk 21]
```diff
@@ -156,358 +191,372 @@ msgid ""
...
 
 #: src/MainWindow.vala:187
 msgid "DuckDuckGo Search"
-msgstr ""
+msgstr "Cerca amb DuckDuckGo"
 
 #: src/MainWindow.vala:193
 msgid "Reset Preferences…"
-msgstr ""
+msgstr "Restabliu les preferències…"
```
Comment (fitojb) [inline po/ca.po]: Reinicialitza (_tú_, not _vós_)

File: po/ca.po [hunk 22]
```diff
@@ -156,358 +191,372 @@ msgid ""
...
 
 #: src/Dialogs/ExternalDialog.vala:28 src/Dialogs/ExternalDialog.vala:30
 msgid "Open Externally?"
-msgstr ""
+msgstr "Voleu obrir externalment?"
```
Comment (fitojb) [inline po/ca.po]: _obrir → obrir-ho_

File: po/ca.po [hunk 23]
```diff
@@ -156,358 +191,372 @@ msgid ""
...
 
 #: src/Dialogs/ExternalDialog.vala:39
 msgid "This page is trying to open an app."
-msgstr ""
+msgstr "Esta pàgina tracta d'obrir una aplicació."
 
 #: src/Dialogs/ExternalDialog.vala:42
 msgid "Your data may not be kept private by the opened app."
-msgstr ""
+msgstr "Les vostres dades podrien no ser manteses en privat per l'aplicació oberta."
```
Comment (fitojb) [inline po/ca.po]: Please rewrite this so it’s active voice instead of passive

File: po/ca.po [hunk 24]
```diff
@@ -156,358 +191,372 @@ msgid ""
...
 
 #: src/Dialogs/PreferencesDialog.vala:33
 msgid "Never Mind"
-msgstr ""
+msgstr "No importa"
 
 #: src/Dialogs/PreferencesDialog.vala:36
 msgid "Reset Preferences"
-msgstr ""
+msgstr "Restabliu les preferències"
```
Comment (fitojb) [inline po/ca.po]: _Reinicialitza_

File: po/ca.po [hunk 25]
```diff
@@ -516,111 +565,121 @@ msgid ""
...
 
 #: src/Widgets/DefaultInfoBar.vala:46
 msgid "Never Ask Again"
-msgstr ""
+msgstr "No torneu a preguntar"
 
 #: src/Widgets/DefaultInfoBar.vala:52
 msgid "Set as Default"
-msgstr ""
+msgstr "Establir per defecte"
```
Comment (fitojb) [inline po/ca.po]: _Establir → Estableix_

File: po/ca.po [hunk 26]
```diff
@@ -516,111 +565,121 @@ msgid ""
...
 
 #: src/Widgets/NativeInfoBar.vala:31
 msgid "Ephemeral is a paid app designed for elementary OS."
-msgstr ""
+msgstr "Ephemeral és una aplicació de pagament disenyada per elementary OS."
```
Comment (fitojb) [inline po/ca.po]: _per → per a_

File: po/ca.po [hunk 27]
```diff
@@ -516,111 +565,121 @@ msgid ""
...
+"És posible que algunes característiques no funcionen correctament "
+"a altres sistemes o entorns."
 
 #: src/Widgets/NativeInfoBar.vala:33
 msgid ""
 "Ephemeral is also typically funded by elementary AppCenter purchases. "
 "Consider donating if you find value in using Ephemeral on other platforms."
 msgstr ""
+"Ephemeral és finançat per les compres a l'AppCenter. Si trobeu "
+"valor en Ephemeral en altrs plataformes, considereu fer una donació."
```
Comment (fitojb) [inline po/ca.po]: Typo: _altrs_ → _altres_

File: po/ca.po [hunk 28]
```diff
@@ -516,111 +565,121 @@ msgid ""
...
 
 #: src/Widgets/NativeInfoBar.vala:41 src/Widgets/PaidInfoBar.vala:43
 msgid "Dismiss"
-msgstr ""
+msgstr "Descarteu"
```
Comment (fitojb) [inline po/ca.po]: _Descarteu → Descarta_

File: po/ca.po [hunk 29]
```diff
@@ -516,111 +565,121 @@ msgid ""
...
 
 #: src/Widgets/NetworkInfoBar.vala:32
 msgid "Connect to the Internet to browse the Web."
-msgstr ""
+msgstr "Connecteu-vos a Internet per navegar per la Xarxa."
 
 #: src/Widgets/NetworkInfoBar.vala:38
 msgid "Never Warn Again"
-msgstr ""
+msgstr "No torneu a avisar"
```
Comment (fitojb) [inline po/ca.po]: _No torneu a avisar → No em tornis a avisar_

File: po/ca.po [hunk 30]
```diff
@@ -516,111 +565,121 @@ msgid ""
...
 #: src/Widgets/PaidInfoBar.vala:32
 msgid "Ephemeral is a paid app"
-msgstr ""
+msgstr "Ephemeral és una aplicació de pagament"
 
 #. TRANSLATORS: This continues the previous string, with terminating punctuation
 #: src/Widgets/PaidInfoBar.vala:34
 msgid "funded by AppCenter purchases."
-msgstr ""
+msgstr "finançat per les compres a AppCenter"
```
Comment (fitojb) [inline po/ca.po]: _finançat → finançada_ (l’aplicació)

File: po/ca.po [hunk 31]
```diff
@@ -516,111 +565,121 @@ msgid ""
...
 
 #: src/Widgets/PaidInfoBar.vala:35
 msgid "Consider purchasing or funding if you find value in using Ephemeral."
-msgstr ""
+msgstr "Si ha trobat profitós Ephemeral, consideri comprar-ho o finançar-ho"
```
Comment (fitojb) [inline po/ca.po]: _vós_ instead of _vostè_

File: po/ca.po [hunk 32]
```diff
@@ -516,111 +565,121 @@ msgid ""
...
 
 #: src/Widgets/UrlEntry.vala:37
 msgid "Enter a URL or search term"
-msgstr ""
+msgstr "Escriviu una adreça o cerqueu un terme"
 
 #: src/Widgets/UrlEntry.vala:43
 msgid "Go"
-msgstr ""
+msgstr "Aneu"
```
Comment (fitojb) [inline po/ca.po]: _Aneu → Vés-hi_

**Comment by capitantrueno** [comment]
> There have been a lot of changes in master since this was filed. Could you resolve conflicts?

I have been very busy last weeks. I will review all changes and I will update strings this weekend (I hope).
Thanks!

**Comment by capitantrueno** [comment]
Reviewed and updated the translated strings

**Comment by capitantrueno** [comment]
> There have been a lot of changes in master since this was filed. Could you resolve conflicts?

Done!

**Comment by capitantrueno** [comment]
Many changes in the branch. It is better to close this pull request and create a new one.

**Pull request closed without merging**